p2p/p2p.go

```diff
@@ -99,86 +114,193 @@ func (s *Service) Stop() error {
 	return nil
 }
 
-// MsgRecPoll starts polling the msgRec channel for any blocks
-func (s *Service) MsgRecPoll(e chan error) {
+// broadcastStatusMessages starts broadcasting status messages
+func (s *Service) broadcastStatusMessages() {
+
+	// Wait for node to finish starting
+	time.Sleep(time.Second)
+
+	// TODO: use generated message
+	msg := statusMessage
+
+	// send message to each connected peer
+	s.host.broadcast(msg)
```

Comment:
we don't broadcast status messages, we send point-to-point them upon connection to a peer

Reply:
i think doing it like this may cause us not to be able to connect to peers. since it's 5 mins between status message sends, the peers will definitely disconnect from us, since they are expecting it within a smaller time period (<1 min most likely, will need to confirm)

Reply:
Correct, status messages need be sent whenever a peer is connected. This only send status messages to connected peers when a new service is started and then every 5 minutes thereafter.